src/overlays/actors/ovl_Obj_Jgame_Light/z_obj_jgame_light.c

```diff
@@ -18,6 +18,11 @@ typedef enum {
     /* 0x2 */ OBJJGAMELIGHT_INCORRECT,
 } ObjJgameLightSignal;
 
+typedef enum {
+    /* 0x0 */ OBJJGAMELIGHT_TORCH_OFF,
+    /* 0x1 */ OBJJGAMELIGHT_TORCH_ON,
+} ObjJgameLightTorchStatus;
```

Comment:
Probably don't need a full enum. Could just use `true/false` and name the variable `isOn`